### DISCUSSION TITLE
error 错误：Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/_libs/interval.cpython-310-darwin.so' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64'))

### DISCUSSION THREAD

**Discussion by fing2022** [binary-husky/gpt_academic#432] [Q&A]
m1pro 电脑 运行main.py后 报错：Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/_libs/interval.cpython-310-darwin.so' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64'))，这种如何解决

**Comment by binary-husky** [comment]
arm架构的cpu吗，抱歉手头上没有arm的电脑

**Reply by fing2022** [reply]
谢谢您
